.github/workflows/formatter.yml

```diff
@@ -0,0 +1,11 @@
+on: push
+name: Code Format Checker
+jobs:
+  lint:
+    name: Code Format Checker
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checker
+      uses: kumagi/clang-format-action@check-diff
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Also I think the token is no longer needed as the action now just clones the repo.

Reply:
Removed.